lib/internal/bootstrap/loaders.js

```diff
@@ -245,8 +229,8 @@ NativeModule.requireWithFallbackInDeps = function(request) {
 };
 
 const getOwn = (target, property, receiver) => {
-  return ReflectApply(ObjectHasOwnProperty, target, [property]) ?
-    ReflectGet(target, property, receiver) :
+  return Reflect.apply(ObjectPrototype.hasOwnProperty, target, [property]) ?
```

Comment:
This is using the runtime mutable lookup of the properties `Reflect['apply']` and `ObjectPrototype['hasOwnProperty']` after they might have been replaced, those methods need to be pulled off to their own references instead of using mutable lookups.

Reply:
@bmeck As @devsnek pointed out, that is not actually the case here.

Reply:
How can `primordials.Reflect` or `primordials.ObjectPrototype` get replaced or mutated by users?

Reply:
ah, disregard this one